namespaces/api-namespace.md

```diff
@@ -0,0 +1,133 @@
+# API Namespace
+
+This document extends [Refract][] [Resource Namespace][] to define REST API data structure elements.
+
+## Content
+
+<!-- TOC depth:2 withLinks:1 updateOnSave:1 -->
+- [API Namespace](#api-namespace)
+	- [Content](#content)
+	- [About this Document](#about-this-document)
+	- [Category (Element)](#category-element)
+	- [Copy (Element)](#copy-element)
+<!-- /TOC -->
+
+## About this Document
+
+This document conforms to [RFC 2119][], which says:
+
+> The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+[MSON][] is used throughout this document.
+
+## Category (Element)
+
+Grouping element – a set of elements forming a logical unit of an API such as
+group of related resources or data structures.
+
+A category element MAY include additional classification of the category.
+The classification MAY hint what is the content or semantics of the category.
+The classification MAY be extended and MAY contain more than one classes.
+
+For example a `category` element may be classified both as `resourceGroup` and
+`dataStructures` to denote it includes both resource and data structures.
+
+### Properties
+
+- `element`: category (string, fixed)
+- `meta`
+    - `class` (array, fixed)
+        - (enum[string])
+            - api - Category is a API top-level group.
+            - resourceGroup - Category is a set of resource.
```

Comment:
@pksunkara yes, that why this is a `category` element – a generic grouping element, not a `resource group` element. `class` serves just an additional classification of the element, but the semantically important information here is `category` – any tool that knows how to deal with a category should be able to understand the element, some tools, can use the additional taxonomy of the element to provide some finer features, but in reality it should be really about `category`